examples/wearable/UIComponents/contents/controls/index.html

```diff
@@ -25,11 +25,6 @@ <h2 class="ui-title">
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
-				<li>
-					<a href="selector/index.html">
```

Comment:
I can not find usage of this file. How about moving it to etc or removing if we don't need it.

Reply:
This link was not working. That is why I removed it. There is no selector folder
https://github.com/Samsung/TAU/tree/tau_1.2/examples/wearable/UIComponents/contents/controls

Reply:
That's clear now. Thanks.